lib/manager/components/NotesViewer.js

```diff
@@ -107,6 +86,13 @@ export default class NotesViewer extends Component<Props, State> {
                           <small title={noteDate.format('h:MMa, MMM. DD YYYY')}>
                             commented {noteDate.fromNow()}
                           </small>
+                          {note.adminOnly &&
+                            <span
+                              title={this.messages('adminOnly')}
+                              className='pull-right'>
+                              <Icon type='lock' />
+                            </span>
+                          }
```

Comment:
This content should be first under the `<Heading>` tag to avoid spilling on narrow displays.
![image](https://user-images.githubusercontent.com/56846598/128401180-633519b0-d622-4f62-88be-ded2b7ef8ff1.png)

Reply:
Do we need the extra `<span>` that wraps around the padlock icon?

Reply:
Both great catches.